assets/scss/_variables_project.scss

```diff
@@ -40,15 +40,18 @@ $table-bg: $secondary-lightest-background;
 $td-enable-google-fonts: false;
 
 /*
- * Replace the default font with Inter.
+ * The $font-family-sans-serif definition here overrides the default value set by docsy
+ * (https://github.com/matrix-org/docsy/blob/66a4e61d2d34edc7196b9df83a7d09cd4af14b47/assets/scss/_variables.scss#L68)
+ * and:
  *
- * The $font-family-sans-serif definition here overrides the default value set by docsy:
- * https://github.com/matrix-org/docsy/blob/66a4e61d2d34edc7196b9df83a7d09cd4af14b47/assets/scss/_variables.scss#L68
- * and adds "Inter" to the front.
+ * - Adds "Inter" to the front (making it the default)
+ * - Inserts "Arial Unicode MS" before "Arial" (because the former contains more mathematical glyphs)
  *
- * The font itself is loaded via stylesheet link layouts/partials/hooks/head-end.html.
+ * The Inter font itself is loaded via stylesheet links layouts/partials/hooks/head-end.html.
  */
-$font-family-sans-serif: "Inter", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+$font-family-sans-serif: "Inter", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
```

Comment:
Isn't -apple-system a deprecated name for system-ui nowadays supported by ~everyone?

- https://caniuse.com/?search=system-ui

Reply:
As far as I understand, we simply "inherit" this from upstream. Would system-ui help fix #1125?

Reply:
That would depend on what the client browser considered as the system font, but having so exotic characters, personally I would attempt to request known good fonts such as Noto.

Reply:
Noto Sans Math [supports these symbols](https://fonts.google.com/noto/specimen/Noto+Sans+Math/tester), and looks as I would expect:

![Screenshot from 2024-08-12 10-13-27](https://github.com/user-attachments/assets/0a420fd4-ac98-49e5-b223-c07db42dd96e)

I suppose the downside is that some browsers may need to download the font instead of using a system one.

Reply:
Maybe we add both Noto and Arial Unicode in the CSS list then?